packages/ui/src/theme/css/component/toggleButtonGroup.scss

```diff
@@ -0,0 +1,22 @@
+.amplify-togglebuttongroup {
+  align-items: var(--amplify-components-togglebuttongroup-align-items);
+  align-content: var(--amplify-components-togglebuttongroup-align-content);
+  justify-content: var(--amplify-components-togglebuttongroup-justify-content);
+  gap: 0;
+}
+
+.amplify-togglebuttongroup {
+  .amplify-togglebutton {
+    &:not(:first-of-type) {
+      border-start-start-radius: 0;
```

Comment:
A fallback for border logical properties is needed for Safari.
see: https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/css/component/fieldGroup.scss#L5-L10